cosmos-hive-auth-provider/src/main/java/com/telefonica/iot/cosmos/hive/authprovider/OAuth2AuthenticationProviderImpl.java

```diff
@@ -39,7 +39,8 @@
  */
 public class OAuth2AuthenticationProviderImpl implements PasswdAuthenticationProvider {
     
-    private static final Logger LOGGER = Logger.getLogger(HttpClientFactory.class);
+    private static final Logger LOGGER = Logger.getLogger(OAuth2AuthenticationProviderImpl.class);
+    private static final OAuth2Cache CACHE = new OAuth2Cache("/home/hive/oauth2.cache");
```

Comment:
It is ugly to have pathfile hardwired in source code... although maybe it doesn't worth the effort to put it outside, i.e. in configuration files.